### PR TITLE
Make @types/[type] work for non-dx types

### PR DESCRIPTION
**Pull request by jaroel**
See also #150

### CODE REVIEW

File: src/plone/restapi/types/utils.py [hunk 1]
```diff
@@ -129,10 +129,16 @@ def get_jsonschema_for_fti(fti, context, request, excluded_fields=None):
...
         excluded_fields = []
 
-    schema = fti.lookupSchema()
-    additional_schemata = tuple(getAdditionalSchemata(portal_type=fti.id))
-
-    fieldsets = get_fieldsets(context, request, schema, additional_schemata)
+    try:
+        schema = fti.lookupSchema()
+    except AttributeError:
+        schema = None
```
Comment (davisagli) [inline src/plone/restapi/types/utils.py]: Maybe add a comment here to remind us when we expect this to happen.

**Comment by jaroel** [comment]
Fair enough 👍

File: src/plone/restapi/types/utils.py (hunk 1 above)
Comment (tisto) [reply src/plone/restapi/types/utils.py]: @jaroel +1 ^^^

File: CHANGES.rst [hunk 2]
```diff
@@ -10,6 +10,9 @@ Changelog
 - Disallow None and u'' when TextLine is required. Refs #351.
   [jaroel]
 
+- Make getting '/@types/Plone Site' work.
```
Comment (tisto) [inline CHANGES.rst]: @jaroel maybe "Make @types endpoint work on portal root"?

**Comment by jaroel** [comment]
I've added some comments and a test for this.
Should be good?

File: src/plone/restapi/tests/test_types.py [hunk 3]
```diff
@@ -108,6 +108,20 @@ def test_get_jsonschema_for_fti(self):
         )
         self.assertNotIn('title', jsonschema['properties'].keys())
 
+    def test_get_jsonschema_for_fti_non_dx(self):
```
Comment (tisto) [inline src/plone/restapi/tests/test_types.py]: @jaroel any reason why you didn't add a test here that just calls @types/Plone%20Site (I guess it kind of sucks that we use blanks in our type ids)

Comment (jaroel) [reply src/plone/restapi/tests/test_types.py]: No reason. Using the Plone Site FTI is a beter idea. I've changed the test to reflect that.

**Comment by tisto** [comment]
Ready to merge if Travis is green.

**Pull request merged**